website/content/accessibility/deaf-and-hard-of-hearing/conference-best-practices.md

```diff
@@ -20,15 +20,15 @@ To make the process as straightforward as possible, we've listed as much as we c
 
 ### DEIA: Diversity + Equity + Inclusion + Accessibility
 
-We encourage you to include the A in your DEI efforts: **Diversity + Equity + Inclusion + Accessibility (DEIA)** — too often our is community an afterthought. We want to change that and need your support!
+We encourage you to include the A in your DEI efforts: **Diversity + Equity + Inclusion + Accessibility (DEIA)** — too often our community _is_ an afterthought. We want to change that and need your support!
 
 #### Disabilities coordinator
 
 We recommend appointing a designated disabilities coordinator to handle any prompt communications involving accommodations before and during the conference. This includes communication with attendees, speakers, facilities, and other event organizers. For signing attendees specifically, the coordinator typically:
-- Facilitates communications between interpreters, assigns them to deaf attendees and location, solicits feedback, and determines fit (e.g., some deaf people may not like the interpreter's signing style, clarity, or quality).
+- Facilitates communications between interpreters, assigns them to deaf attendees and location, solicits feedback, and determines the fit (e.g., some deaf people may not like the interpreter's signing style, clarity, or quality).
 - Ensures proper lighting and interpreter staging.
-- Provide extra equipment in case of failures, such as lights, stands, chairs, etc.
-- Takes attendee feedback on interpreters
+- Provides extra equipment in case of failures, such as lights, stands, chairs, etc.
+- Takes attendees feedback on interpreters
```

Comment:
Attendee feedback is actually correct. Not sure how to explain it grammatically, but in these cases, the first noun is always singular. Maybe the right way to phrase it is that the first noun acts as a classifier for the second noun. If plural, the second noun is plural only. Feedback doesn't have plural, though, so it looks the same as if it were singular. Takes attendee viewpoints would be an example where the plural of the second noun is visible.

Reply:
Agree. I wasn't 100% sure on this. Thanks for the excellent clarification. Changing this back to `attendee` @CathPag